ui/main/src/app/business/services/opfabAPI.service.ts

```diff
@@ -246,12 +246,6 @@ export class OpfabAPIService {
         opfab.currentCard.getEntitiesAllowedToRespond = function () {
             return self.currentCard.entitiesAllowedToRespond;
         };
-        opfab.currentCard.getEntityUsedForUserResponse = function () {
-            logger.warn(
-                ' WARNING : Use of opfab.currentCard.getEntityUsedForUserResponse is deprecated, you should use opfab.currentCard.getEntitiesUsableForUserResponse instead'
-            );
-            return self.currentCard.entityUsedForUserResponse;
```

Comment:
You have as well to remove the "entityUsedForUserResponse" variable in currentCard  and the settings of the value in card-body.components.ts as it is not needed anymore

As a consequence, it should be possible as well to remove the global variable userEntityIdToUseForResponse in card-body-components.ts :smiley:

Reply:
Done.